crates/bevy_ecs/src/reflect/entity_commands.rs

```diff
@@ -0,0 +1,318 @@
+﻿use crate::system::{Command, EntityCommands};
+use crate::{entity::Entity, reflect::ReflectComponent, world::World};
+use bevy_reflect::{Reflect, TypeRegistry};
+
+/// An extension trait for [`EntityCommands`] for reflection related functions
+pub trait EntityCommandsReflectExtension {
+    /// Inserts the given boxed reflect component to the entity using the reflection data in the supplied
+    /// [`TypeRegistry`]. This will overwrite any previous component of the same type.
```

Comment:
Nit: The first line should be more of a quick summary (1–2 sentences). Additional details can go in the following paragraphs.
```suggestion
    /// Inserts the given boxed reflect component to the entity using the reflection data in the supplied
    /// [`TypeRegistry`].
    ///
    /// This will overwrite any previous component of the same type.
```